build.gradle

```diff
@@ -71,6 +72,31 @@ allprojects {
       downloadSources = false
     }
   }
+
+  plugins.withId('net.ltgt.errorprone') {
+    plugins.apply('net.ltgt.nullaway')
+    dependencies {
+      errorprone "com.google.errorprone:error_prone_core"
+      errorprone "com.uber.nullaway:nullaway"
+    }
+
+    nullaway {
+      annotatedPackages.addAll("io.opentelemetry", "com.linecorp.armeria,com.google.common")
+    }
+
+    tasks.withType(JavaCompile).configureEach {
+      if (!name.toLowerCase().contains("test")) {
+        options.errorprone {
+          nullaway {
+            severity = net.ltgt.gradle.errorprone.CheckSeverity.ERROR
```

Comment:
I had a similar problem before, but AFAICT, there's no way to access plugin classes from a Gradle "script plugin", which is what `apply from: *.gradle` type of stuff is. So I don't see a way to do this sort of configuration from `java.gradle` and just moved it here.